examples/0006-reuse-base-path.toml

```diff
@@ -0,0 +1,25 @@
+[relaychain]
```

Comment:
Maybe we can add an small docs on top of this network config describing the example (e.g in this case the use of the `base_path` )
Thx!